deploy/helm/sumologic/templates/setup/job.yaml

```diff
@@ -52,6 +52,7 @@ spec:
       containers:
       - name: setup
         image: {{ .Values.sumologic.setup.job.image.repository }}:{{ .Values.sumologic.setup.job.image.tag }}
+        imagePullSecrets: {{ .Values.sumologic.setup.job.image.pullSecrets }}
```

Comment:
What about this? https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v2.19.0/deploy/docs/Working_with_container_registries.md?plain=1#L46